hunts/views.py

```diff
@@ -144,6 +144,26 @@ def redirect_to_drive(request, hunt_slug):
         return redirect("/")
 
 
+@login_required(login_url="/")
+def sync_discord_roles(request, hunt_slug):
+    if request.method != "POST":
+        return HttpResponseForbidden()
+    if not request.user.is_staff:
+        return HttpResponseForbidden()
+
+    hunt = Hunt.get_object_or_404(user=request.user, slug=hunt_slug)
+
+    import chat.tasks
+
+    chat.tasks.sync_roles(
+        hunt,
+        settings.CHAT_DEFAULT_SERVICE,
+    )
```

Comment:
I think this will run this synchronously by the web process, right? Need to call `.delay()` if you want to create a task